theme/theme.go

```diff
@@ -123,6 +128,11 @@ const (
 	// Since 2.0.0
 	SizeNameText fyne.ThemeSizeName = "text"
 
+	// SizeNameEntryUnderline is the name of theme lookup for widget.Entry underline size.
+	//
+	// Since 2.0.0
+	SizeNameEntryUnderline fyne.ThemeSizeName = "entryUnderline"
```

Comment:
I could not think of a more generic way to describe this. Perhaps someone else can?
Or would we be better off hard coding it in Entry as `Padding()/2` like some other widgets do?

Reply:
I don't like the idea of hardcoding the size with padding as those are completely different sizes (they are not related).